### DISCUSSION TITLE
TSC members assigned to strategic initiatives

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#53] [General]
In our most recent TSC meeting it was proposed that each FINOS strategic initiative (FDC3, OSR, RegTech) has a point-person assigned from the TSC, working alongside the initiative manager (FINOS team member).

Their role would be TSC representation; guiding technical decision making, and bringing in the TSC if and when it is needed. 

This is a thread for further discussion on this topic, and if anyone wants to 'hand raise' for a specific initiative, please do!

**Comment by ColinEberhardt** [comment]
I am certainly in favour of this approach :-)

**Comment by stephengoldbaum** [comment]
I'm also in favor.  In addition, I propose that all Special Interest Groups and key projects have a mechanism to send important updates to the TSC.  Some of the ideas mentioned were CCing or mentions when relevant.
